extensions/web/lib.deno_web.d.ts

```diff
@@ -205,6 +205,28 @@ declare class TextEncoder {
   readonly [Symbol.toStringTag]: string;
 }
 
+declare class TextDecoderStream {
+  /** Returns encoding's name, lowercased. */
+  readonly encoding: string;
+  /** Returns `true` if error mode is "fatal", and `false` otherwise. */
+  readonly fatal: boolean;
+  /** Returns `true` if ignore BOM flag is set, and `false` otherwise. */
+  readonly ignoreBOM = false;
+  constructor(
+    label?: string,
+    options?: { fatal?: boolean; ignoreBOM?: boolean },
```

Comment:
In the `lib.dom.d.ts` this is broken out as a seperate interface of:

```ts
interface TextDecoderOptions {
    fatal?: boolean;
    ignoreBOM?: boolean;
}
```

It would be good to align to that.